clients/da-vinci-client/src/test/java/com/linkedin/davinci/kafka/consumer/StoreBufferServiceTest.java

```diff
@@ -213,7 +224,7 @@ public void testGetDrainerIndexForConsumerRecord(boolean queueLeaderWrites) {
 
   @Test(dataProviderClass = DataProviderUtils.class, dataProvider = "True-and-False")
   public void testRunWhenThrowVeniceCheckSumFailException(boolean queueLeaderWrites) throws Exception {
```

Comment:
Do we not want to verify any of the new metrics in this test class? e.g. `recordInternalProcessingError`

Reply:
IMO, we don't have to add many tests for `stats` and usually they get tested as a part of business logic. I will make sure they are emitted correctly after deploying to per-production. That being said, I've added some tests for SBS stats. Please take a look @xunyin8

Reply:
Right, we don't have to add **many** tests for stats but we should at least have one smoke test and most of the time it's not to verify the new stats are added correctly but prevent regressions that break stats. We've had numerous incidents where a change accidentally broke some stats.

Reply:
Acked. I will see if I can add more on this in my next PR. The SBS is easy to add but for SIT, something was not working for me so I decided to defer it.

Reply:
I still couldn't get it working.. Let me merge this one for now and we can add it later.